app/views/trainees/diversity/disability_disclosures/edit.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= content_for(:breadcrumbs) do %>
+  <%= render_component GovukComponent::BackLink.new(
+    text: "Back",
+    href: request.referrer
```

Comment:
We need to be explicit with the path and not infer form the client.

It broken now

https://dfe-twd-rttd-pr-124.herokuapp.com/trainees/3/diversity/disability-disclosure/edit

Reply:
Forgot to clean this up thanks for spotting. I set that for testing 🙈

Reply:
fixed in https://github.com/DFE-Digital/register-trainee-teachers/pull/124/commits/c538876731f3309572ed3562fb85e67cf0347b98